psi/app/__init__.py

```diff
@@ -85,11 +86,7 @@ def init_db(flask_app):
 
 
 def init_migrate(flask_app, database):
```

Comment:
At this point, this function isn't necessary, but I wanted to keep this pull request short